### PR TITLE
Replace old conversation options dialog with a bottom sheet

### PR DESCRIPTION
**Pull request by JPonte**
#### APK
[Download build #10524](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10524/artifact/build/artifact/wire-dev-PR1467-10524.apk)
[Download build #10529](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10529/artifact/build/artifact/wire-dev-PR1467-10529.apk)
[Download build #10533](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10533/artifact/build/artifact/wire-dev-PR1467-10533.apk)
[Download build #10544](http://192.168.120.33:8080/job/Pull%20Request%20Builder/10544/artifact/build/artifact/wire-dev-PR1467-10544.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/conversationlist/ConversationListManagerFragment.scala [hunk 1]
```diff
@@ -387,23 +386,18 @@ class ConversationListManagerFragment extends Fragment
...
-        case _ if pickUserController.isShowingPickUser(getCurrentPickerDestination) =>
-          pickUserController.hidePickUser(getCurrentPickerDestination)
-          true
-        case _ => false
-      }
+    withBackstackHead {
+      case Some(f: IntegrationDetailsFragment) if f.onBackPressed() => true
+      case Some(f: SearchUIFragment) if f.onBackPressed() => true
+      case Some(f: ArchiveListFragment) if f.onBackPressed() => true
+      case Some(f: NewConversationFragment) if f.onBackPressed() => true
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/conversationlist/ConversationListManagerFragment.scala]: You should just be able to do `Some(f: FragmentHelper) if f.onBackPressed() => true` and you're done

File: app/src/main/scala/com/waz/zclient/participants/fragments/ParticipantFragment.scala [hunk 2]
```diff
@@ -183,17 +179,10 @@ class ParticipantFragment extends BaseFragment[ParticipantFragment.Container] wi
     }
   }
 
-  override def onShowEditConversationName(show: Boolean): Unit =
-    bodyContainer.foreach { view =>
-      if (show) ViewUtils.fadeOutView(view)
-      else ViewUtils.fadeInView(view)
-    }
+  override def onShowEditConversationName(show: Boolean): Unit = {}
```
Comment (deanrobertcook) [inline app/src/main/scala/com/waz/zclient/participants/fragments/ParticipantFragment.scala]: Can we just delete the method

**Comment by JPonte** [comment]
build again

**Comment by JPonte** [comment]
build again

**Pull request merged**